modules/vba_documents/app/mailers/vba_documents/views/unsuccessful_report.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Unsuccessful Submissions </h1>
+<hr />
+<h3>From <%= @date_form %> -- <%= @date_to %>
```

Comment:
This is missing the closing </h3> tag

Reply:
rip thanks :P

Reply:
resolved in https://github.com/department-of-veterans-affairs/vets-api/pull/3429/commits/88f0a6cb1a39d9cee2bb1e56f0383937ff773d80